salt/modules/kapacitor.py

```diff
@@ -154,6 +159,12 @@ def define_task(name,
 
         salt '*' kapacitor.define_task cpu salt://kapacitor/cpu.tick database=telegraf
     '''
+    if not database and not dbrps:
```

Comment:
@cachedout Since we cannot change the parameter directly from optional to mandatory to avoid breaking anything, what do you think about this approach? :thinking:

Reply:
If we do this we should also log an error with `log.error()` IMHO